File: add/codesnippet/CSharp/5d3f62ad-c21a-4506-bd52-_1.cs

            // Create a ClientTarget object.
            clientTarget = new ClientTarget(
              "my alias", "My User Agent");

            // Remove it from the collection
            // (if exists).
            clientTargets.Remove(clientTarget);

            // Update the configuration file.
            if (!clientTargetSection.IsReadOnly())
              configuration.Save();

File: add/codesnippet/VisualBasic/5d3f62ad-c21a-4506-bd52-_1.vb
                    ' Create a ClientTarget object.
                    clientTarget = New ClientTarget( _
                    "my alias", "My User Agent")

                    ' Remove it from the collection
                    ' (if exists).
                    clientTargets.Remove(clientTarget)

                    ' Update the configuration file.
                    If Not clientTargetSection.IsReadOnly() Then
                        configuration.Save()
                    End If